scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/utils/Encrypt.scala

```diff
@@ -101,6 +104,9 @@ object Encrypt extends Supportive {
       opt[String]('h', "header")
         .action((x, c) => c.copy(header = x))
         .text("whether to write header to the csv file, default is false")
+      opt[Int]('p', "partition")
```

Comment:
I think you can provide a default partition number, like ( file size / the HDFS block size) + 1